test/src/edu/stanford/nlp/util/IntTupleTest.java

```diff
@@ -0,0 +1,93 @@
+package edu.stanford.nlp.util;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class IntTupleTest {
+  IntTuple sut6Tuple;
+  IntTuple sut1Tuple;
+
+  @Before
+  public void setup() {
+    int[] nrs = {1, 2, 3, 5, 7, 11};
+    this.sut6Tuple = new IntTuple(nrs);
+    this.sut1Tuple = new IntTuple(13);
+  }
+
+  @Test
+  public void testCompareTo() {
```

Comment:
Maybe a few comparisons of tuples of the same size would be of use as well.  Checks on tuples of different lengths where the initial item lists are the same and the sort is based on the length of the tuple.  Etc etc

Reply:
Thanks @AngledLuffa I have increased the coverage of that case so that more cases/branches are tested.